.github/workflows/main.yml

```diff
@@ -4,9 +4,11 @@ on:
   push:
     branches:
       - $default-branch
+      - brady/add-aarch64-ci
```

Comment:
I need to look into this a little more. Apparently `$default-branch` doesn't work for triggers, even though I pulled from several example that showed it works.

At worst, we can hard code it to the default branch  (i.e. `1.67.1`) and updated it when someone here updates the default branch. I'll investigate to see if there's way to make that automatic here so that it "just works".

Reply:
That's... weird.

Reply:
Yeah. I am just going to do it by name for now so that you and @thomcc have something to use immediately. I can figure out the default branch stuff later.